baseclass-samples/java-callouts-baseclass-exception/callout/src/main/java/BaseclassException.java

```diff
@@ -0,0 +1,26 @@
+package com.apigeesample;
+
+import com.apigee.flow.execution.ExecutionContext;
+import com.apigee.flow.execution.ExecutionResult;
+import com.apigee.flow.execution.spi.Execution;
+import com.apigee.flow.message.MessageContext;
+import com.google.apigee.calloutbaseclass.CalloutBase;
+
+import java.util.Map;
+
+public class BaseclassException extends CalloutBase implements Execution {
```

Comment:
Let's add a preamble comment that describes what this class does, its purpose, etc..  Naming-wise this class is not an exception nor a base class.  I think we need to pick a better name.  Java coder will think this class inherits from Exception or Throwable.  For example, there is IOException.